<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7653" started-at="2019-07-22T05:01:58Z" finished-at="2019-07-22T05:02:06Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7653" started-at="2019-07-22T05:01:58Z" finished-at="2019-07-22T05:02:06Z">
      <class name="exentreportlistener.ExtentReportListener">
        <test-method status="PASS" signature="startReport(java.lang.String, java.lang.String)[pri:0, instance:exentreportlistener.ExtentReportListener@42e26948]" name="startReport" is-config="true" duration-ms="250" started-at="2019-07-22T05:01:58Z" finished-at="2019-07-22T05:01:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[param-val-not-found]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[param-val-not-found]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="FAIL" signature="login()[pri:0, instance:exentreportlistener.ExtentReportListener@42e26948]" name="login" duration-ms="6681" started-at="2019-07-22T05:01:58Z" finished-at="2019-07-22T05:02:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Google] but found [Srinivas]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Google] but found [Srinivas]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at exentreportlistener.ExtentReportListener.login(ExtentReportListener.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:exentreportlistener.ExtentReportListener@42e26948]" name="getResult" is-config="true" duration-ms="364" started-at="2019-07-22T05:02:05Z" finished-at="2019-07-22T05:02:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=ExtentReportListener.login()[pri:0, instance:exentreportlistener.ExtentReportListener@42e26948] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:exentreportlistener.ExtentReportListener@42e26948]" name="tearDown" is-config="true" duration-ms="340" started-at="2019-07-22T05:02:06Z" finished-at="2019-07-22T05:02:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- exentreportlistener.ExtentReportListener -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
